main.py

```diff
@@ -21,6 +21,12 @@
 import re
 import logging
 
+import websocket
+try:
+    import thread
+except ImportError:
```

Comment:
As we're on python 3, we should do it properly and not rely on Errors being thrown